start.py

```diff
@@ -1217,14 +1217,13 @@ def restart_callback():
 def start_metrics(m2ee):
     metrics_interval = os.getenv("METRICS_INTERVAL")
     profile = os.getenv("PROFILE")
-    if metrics_interval and profile != "free":
+    if metrics_interval:
```

Comment:
As above, I would change this block to initialize a `FreeAppsMetricsEmitterThread` is `buildpackutil.is_free_app` returns true.